core/src/main/java/org/apache/accumulo/core/spi/crypto/CryptoService.java

```diff
@@ -21,7 +21,7 @@
 import java.util.Map;
 
 /**
- * Self contained cryptographic service. All on disk encryption and decryption will take place
+ * A self-contained cryptographic service. All on disk encryption and decryption will take place
```

Comment:
This comment is outside the scope of the changes here, but I wonder if `CryptoException` should extend `GeneralSecurityException` like the other Exceptions in the crypto code. I'm also wondering if `CryptoException` should also be declared on all the methods and the implementations changed to *not* throw RuntimeExceptions.

Reply:
Yeah, this would be a good follow on task.